src/containers/ListView/index.jsx

```diff
@@ -3,18 +3,27 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
 import {
-  Badge,
   DataTable,
   TextFilter,
+  MultiSelectDropdownFilter,
 } from '@edx/paragon';
 
+import {
+  gradingStatusDisplay,
+} from 'data/services/lms/constants';
+
 import selectors from 'data/selectors';
 import thunkActions from 'data/thunkActions';
 
 import StatusBadge from 'components/StatusBadge';
 import ReviewModal from 'containers/ReviewModal';
 import './ListView.scss';
 
+const gradeStatusFilterChoices = Object.keys(gradingStatusDisplay).map(key => ({
+  name: gradingStatusDisplay[key],
```

Comment:
General convention here is to call this`...Options` as it is populating the options for a select control.